internal/util/vault_tokens.go

```diff
@@ -163,6 +171,97 @@ func (kms *VaultTokensKMS) parseConfig(config map[string]interface{}, secrets ma
 	return nil
 }
 
+// initCertificates updates the kms.vaultConfig with the options from config
+// it calls the kubernetes secrets and get the required data.
+
```

Comment:
not sure if this empty line causes issues with `go doc`. I usually comment an empty line in these cases (no idea what is better)

Reply:
even we are not generating the go doc I think it should be fine for now.